client/my-sites/email/inbox/index.js

```diff
@@ -33,26 +34,31 @@ const InboxManagement = ( { domains } ) => {
 		);
 	}
 
-	const domainsWithSubscriptions = domains.filter(
-		( domain ) => ! domain.isWPCOMDomain && hasPaidEmailWithUs( domain )
-	);
+	// Find out if the site has at least one mailbox in the site
+	const hasOneOrMoreMailboxes = domains
+		.filter( ( domain ) => ! domain.isWPCOMDomain )
+		.reduce( ( accumulator, domain ) => {
+			const mailboxCountForDomain =
+				getGSuiteMailboxCount( domain ) + getConfiguredTitanMailboxCount( domain );
+			accumulator = mailboxCountForDomain > 0;
 
-	//EmailManagementHome logic will handle the case where there is only one domain to show directly the email comparison
-	//and also if there is no domain so, it will show a CTA to buy domain
-	if ( domainsWithSubscriptions.length === 0 ) {
-		return (
-			<CalypsoShoppingCartProvider>
-				<EmailManagementHome
-					emailListInactiveHeader={ getMainHeader() }
-					sectionHeaderLabel={ translate( 'Domains' ) }
-					showActiveDomainList={ false }
```

Comment:
I think we want to keep the `showActiveDomainList` for the case where the user doesn't have email services on any domain, even email forwarding.

Maybe we can add something like the following check:
```
const domainsWithEmailServices = domains
	.filter( ( domain ) => 
		! domain.isWPCOMDomain &&
			( hasPaidEmailWithUs( domain ) || hasEmailForwards( domain ) );
	);
const showActiveDomainList = domainsWithEmailServices.length > 0;
```

Reply:
The `hasEmailsForwards` does not look right to me... In this case if user have a forward email, we won't show the possibility to upgrade to a new email provider.